pkgs/development/python-modules/ihatemoney/default.nix

```
@@ -132,6 +132,13 @@ buildPythonPackage rec {
     debts
   ];
 
+  postPatch = ''
+    # upstream performed the update without needing to patch the code
+    # the original patch does not apply, sadly
+    # https://github.com/spiral-project/ihatemoney/pull/912
+    substituteInPlace setup.cfg --replace "Flask-WTF>=0.14.3,<1" "Flask-WTF>=0.14.3,<2"
+  '';
```

comments inside of the code block affect the build. Since there's only one comment I suggest we move it outside of the code block.

```suggestion
  # upstream performed the update without needing to patch the code
  # the original patch does not apply, sadly
  # https://github.com/spiral-project/ihatemoney/pull/912
  postPatch = ''
    substituteInPlace setup.cfg --replace "Flask-WTF>=0.14.3,<1" "Flask-WTF>=0.14.3,<2"
  '';
```